src/main/java/net/greatstart/dto/DtoProject.java

```diff
@@ -13,5 +14,8 @@
     private DtoCategory category;
     @Valid
     private DtoProjectDescription desc;
+    private DtoUserProfile owner;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "owner" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)